src/main/webapp/script.js

```diff
@@ -0,0 +1,114 @@
+
+import {FlyControls} from 'https://threejs.org/examples/jsm/controls/FlyControls.js';
+import {GUI} from 'https://threejs.org/examples/jsm/libs/dat.gui.module.js';
+
+// global objects
+let scene;
+let camera;
+let renderer;
+let clock;
+let controls;
+const rotationData = new Map();
+const points = [];
+
+/**
+ * Intializes the scene and its child objects
+ */
+function init() {
+  scene = new THREE.Scene();
+  camera = new THREE.PerspectiveCamera(
+      75, window.innerWidth / window.innerHeight, .1, 1000);
+  camera.position.set(0, 1, 1);
+  renderer = new THREE.WebGLRenderer();
+  renderer.setSize(window.innerWidth, window.innerHeight);
+  document.body.appendChild(renderer.domElement);
+  clock = new THREE.Clock();
+
+  // Camera controls, imported from FlyControls library
+  controls = new FlyControls(camera, renderer.domElement);
+  controls.movementSpeed = 1;
+  controls.rollSpeed = Math.PI / 10;
+  controls.autoForward = false;
+  controls.dragToLook = true;
+
+  // Scene Objects
+  let geometry = new THREE.PlaneGeometry(5, 25, 1);
+  let material =
+      new THREE.MeshBasicMaterial({color: 'pink', side: THREE.DoubleSide});
+  const plane = new THREE.Mesh(geometry, material);
+  plane.rotation.x = Math.PI / 2;
+  plane.position.z = -10;
+  scene.add(plane);
+
+  // Fake Pose Data
+  material = new THREE.LineBasicMaterial({color: 'red'});
```

Comment:
These areas with comments look like good places to make functions

Reply:
+1